src/uri.ts

```diff
@@ -24,15 +29,16 @@ export function resolveURI(uri: string): ResolvedURI {
     if (!rev) {
         throw new Error('Could not determine revision')
     }
-    const path = url.hash.slice(1)
-    console.log(uri, path)
-    if (!path) {
-        throw new Error('Could not determine file path')
-    }
+    return { repo, rev }
+}
+
+/**
+ * Resolve a URI of the forms git://github.com/owner/repo?rev#path and file:///path to an absolute reference
+ */
+export function resolveFileURI(uri: string): ResolvedFileURI {
```

Comment:
Perhaps name it `resolveDocumentURI` to not give the impression this takes a `file:` URI